src/main/java/com/jcabi/s3/Ocket.java

```diff
@@ -110,7 +110,8 @@ void read(@NotNull(message = "output stream can't be NULL")
     /**
      * Write new content to the object.
      * @param input Where to get content
-     * @param meta Metadata to save
+     * @param meta Metadata to save. Should contains input length for large
+     *  object, it will make multi part upload possible.
```

Comment:
@YamStranger "for large objects, otherwise multi-part uploads won't be possible."